ledger/narwhal/batch-header/src/lib.rs

```diff
@@ -56,20 +56,6 @@ pub struct BatchHeader<N: Network> {
 }
 
 impl<N: Network> BatchHeader<N> {
-    /// The maximum number of certificates in a batch.
-    #[cfg(not(any(test, feature = "test-helpers")))]
-    pub const MAX_CERTIFICATES: u16 = N::MAX_CERTIFICATES;
-    /// The maximum number of certificates in a batch.
-    /// This is deliberately set to a high value (100) for testing purposes only.
-    #[cfg(any(test, feature = "test-helpers"))]
-    pub const MAX_CERTIFICATES: u16 = 100;
-    /// The maximum number of certificates in a batch before consensus V3 rules apply.
-    #[cfg(not(any(test, feature = "test-helpers")))]
-    pub const MAX_CERTIFICATES_BEFORE_V3: u16 = N::MAX_CERTIFICATES_BEFORE_V3;
-    /// The maximum number of certificates in a batch before consensus V3 rules apply.
-    /// This is deliberately set to a high value (100) for testing purposes only.
-    #[cfg(any(test, feature = "test-helpers"))]
-    pub const MAX_CERTIFICATES_BEFORE_V3: u16 = 100;
     /// The maximum number of rounds to store before garbage collecting.
     pub const MAX_GC_ROUNDS: usize = 100;
```

Comment:
nit: in the future we might want to move these final 2 consts to the `Network` too